src/reducers/files-and-folders/files-and-folders-thunks.ts

```diff
@@ -70,7 +72,12 @@ export const moveElement = (elementId, newParentId): ArchifiltreThunkAction => (
 ) => {
   const filesAndFolders = getFilesAndFoldersFromStore(getState());
   const parent = findElementParent(elementId, filesAndFolders);
-
+  if (
+    isExactFileOrAncestor(newParentId, elementId) ||
```

Comment:
Il faut se baser sur le virtualPath et non sur le path pour faire la comparaison. Sinon, tu interdis des comportements autorisés.

Reply:
Steps to reproduce : 
1) Sors un dossier de son parent.
2) Essaie de mettre l'ancien parent dans le dossier que tu as déplacé.

Normalement, ça devrait fonctionner, mais c'est bloqué.